Functional-1/doubling/test/index.test.ts

```diff
@@ -0,0 +1,28 @@
+import { assert, should } from "chai";
+import { doubling } from "../src/index";
+
+/*
+Given a list of integers, return a list where each integer is multiplied by 2.
+
+
+doubling([1, 2, 3]) → [2, 4, 6]
+doubling([6, 8, 6, 8, -1]) → [12, 16, 12, 16, -2]
+doubling([]) → []
+*/
+
+describe("doubling", () => {
+  it("Should return a doubled array: [2, 4, 6] from the given array [1, 2, 3]", () => {
```

Comment:
When it comes to test descriptions, please try to rather phrase them without having to specify exact values.